spectree/plugins/falcon_plugin.py

```diff
@@ -79,7 +79,7 @@ def register_route(self, app: Any):
                 f"/{self.config.path}/{ui}",
                 self.DOC_PAGE_ROUTE_CLASS(
                     self.config.page_templates[ui],
-                    spec_url=self.config.spec_url,
+                    spec_url=self.config.filename,
```

Comment:
`spec_url` is `f"/{self.path}/{self.filename}"`, I don't get why it should only contain the filename. Can you explain in detail?

Reply:
As I wrote to comment - absolute address is /apidoc/openapi.json ; so for example for:
* http://foo.bar => http://foo.bar/apidoc.openapi.json
* http://foo.bar/foo/bar/ => http://foo.bar/apidoc/openapi.json

But if there is some subdirectory for API, for example if multiple APIs is available, then absolute address is not good solution. For example base root dirs (basepaths) for APIs:
* http://foo.bar/foo/bar/ => http://foo.bar/apidoc/openapi.json ====\  **conflict!!!**
* http://foo.bar/bar/foo/ => http://foo.bar/apidoc/openapi.json ====/
Right paths should be:
* http://foo.bar/foo/bar/ => http://foo.bar/foo/bar/apidoc/openapi.json
* http://foo.bar/bar/foo/ => http://foo.bar/bar/foo/apidoc/openapi.json

...If you have set some root directory for API, swagger and redoc try to find file absolutely on /apidoc/openapi.json (not relatively) so it show errors only. It is needed to search openapi.json in set subdirectory.

Reply:
Got it. Thanks.